frontend/sw360-portlet/src/main/webapp/html/utils/includes/usingProjectsTable.jspf

```diff
@@ -9,29 +9,55 @@
 --%>
 
 <core_rt:if test="${usingProjects != null  && usingProjects.size()>0}">
-    <table class="table info_table" title="Usage overview">
+    <table class="table info_table" style="margin-bottom: 0; border-bottom-width: 0">
```

Comment:
This might be difficult because importing the data via javascript implicitly defines the column headers.